lib/options.go

```diff
@@ -225,6 +225,7 @@ type Options struct {
 
 	// Accept invalid or untrusted TLS certificates.
 	InsecureSkipTLSVerify null.Bool `json:"insecureSkipTLSVerify" envconfig:"K6_INSECURE_SKIP_TLS_VERIFY"`
+	LogTLSKey             null.Bool `json:"logTLSKey" envconfig:"K6_LOG_TLS_KEY"`
```

Comment:
This should be a string to a separate file that we log the keys to. Reusing the logger output doesn't seem like a good idea at all

Reply:
On second though you should also move it to [RuntimeOptions](https://github.com/loadimpact/k6/blob/479707f90eae51ee385b3424e6a1259702608e8a/lib/runtime_options.go#L32) so you don't have a way to set it from inside the script, as that is totally what we want to avoid.